sdk/lib/ucrt/inc/corecrt_internal_big_integer.h

```diff
@@ -698,6 +699,21 @@ __forceinline uint32_t __cdecl count_sequential_high_zeroes(uint32_t const u) th
 
             add edx, ecx
         }
+        #else // ^^^ _MSC_VER ^^^ // vvv !_MSC_VER vvv //
+	    uint64_t retval;
+	    __asm__(
+            "mull %[multiplier]\n"
```

Comment:
Shouldn't there be a `movl` before this multiplier?

Reply:
That is already implicit in the constraint `[multiplicand_hi] "a" ((uint32_t)`. The `"a"` says that it goes into eax.